cupy/fallback_mode/__init__.py

```diff
@@ -0,0 +1,6 @@
+# Attributes and Methods for fallback_mode
+# Auto-execute numpy method when corresponding cupy method is not found
+
+# "NOQA" to suppress flake8 warning
+from cupy.fallback_mode.fallback import numpy  # NOQA
+from cupy.fallback_mode.fallback import _RecursiveAttr  # NOQA
```

Comment:
Is this needed?

Reply:
Oh, I forgot to remove that while removing notification support. Will remove that too.